src/test/java/com/softserveinc/ita/vsaroz/PracticeTest.java

```diff
@@ -25,17 +25,20 @@ public void setUp() {
 
     @Test
     public void verifyAddToCartTest () {
-        productElement
+        $x(productElement)
                 .shouldBe(visible)
                 .click();
-        addToCartButtonElement
+        $x(addToCartButtonElement)
                 .shouldBe(visible)
                 .click();
-        closeModalButtonElement
+        $x(closeModalButtonElement)
                 .shouldBe(disappear)
                 .click();
-        viewShoppingCartButtonElement.click();
-        String quantity = itemsInCart.getText();
-        assertThat(quantity).isNotNull();
+        $x(viewShoppingCartButtonElement).click();
+        var quantity = $x(itemsInCart).getText();
+        Integer.parseInt(quantity);
```

Comment:
use static import of the parse method and move into assert, you are parsed string and have not used anywhere